diff --git a/Sources/Foundation/Port.swift b/Sources/Foundation/Port.swift
index c53263f0..987bb12b 100644
--- a/Sources/Foundation/Port.swift
+++ b/Sources/Foundation/Port.swift
@@ -88,7 +88,7 @@ public protocol PortDelegate: AnyObject {
 @available(*, unavailable, message: "SocketPort is not available on this platform.")
 open class SocketPort: Port {}
 
-#else
+#endif
 
 #if canImport(Glibc) && !os(Android) && !os(OpenBSD)
 import Glibc
@@ -106,7 +106,7 @@ fileprivate let INADDR_ANY: in_addr_t = 0
 #if os(OpenBSD)
 fileprivate let INADDR_LOOPBACK = 0x7f000001
 #endif
-#endif
+
 
 
 #if canImport(WinSDK)
